storage/sqlite/src/lib.rs

```diff
@@ -225,26 +225,42 @@ impl ShallowClone for SqliteImpl {}
 
 impl backend::BackendImpl for SqliteImpl {}
 
+#[derive(Eq, PartialEq, Clone, Debug)]
+enum SqliteBackend {
```

Comment:
Hmm `SqliteBackend` may be a bit confusing name. Not that I have a better alternative though. `SqliteStorageMode`? Not sure, and maybe not too important for now, it's not a public type.